### PR TITLE
Merge Feature Branch into Development Branch

### PR DESCRIPTION
**Pull request by RamseyNjire**
In this project I build Rails forms, first using raw HTML, and then using a variety of Rails helpers. Hope you love it!

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    else
+      render :new
+    end
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (addod19) [inline app/controllers/users_controller.rb]: Kindly change `redirect_to new_user_path` to `redirect_to edit_user_path`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Require changs :recycle: :traffic_light: 

Hi @RamseyNjire, great job on this project :+1: Kindly address the comments below so we can approve the project :slightly_smiling_face: 

- Kindly fix the error below. This error came as a result of running `rails s` after running the bundle install and migration. As a result of this, I am not able to test the app. Let us kindly fix it :smile: 

[Optional]
- Kindly create a partial for user form and reuse in `new` and `edit` view
- Kindly create a form view to displays a list of the error messages (with #errors and full_messages usage)
- Validate the email format by using either a regular expression or a built-in URI method.

Once this is fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Comment by RamseyNjire** [comment]
Hey! I've made the change in the user controller.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
```
Comment (peterrobert) [inline app/controllers/users_controller.rb:19]: When the data is valid, You should navigate to the edit_path

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 

HELLO @RamseyNjire  VERY NICE PROGRESS :clap: GREAT WORK IMPLEMENTING THE CHANGES AS REQUESTED. HOWEVER, THERE ARE A FEW CHANGES TO BE MADE KINDLY CHECK BELOW :arrow_down: 

- [ ] There seems to be a problem with webpack in your project kindly fix it :arrow_down: 
Screenshot from 2020-11-23 11-23-46

NB: I was unable to run your application. Kindly fix and submit for another review.

### you can reach me on slack @peterrobert

### HAPPY CODING :computer:

**Comment by RamseyNjire** [comment]
The webpacker issue has been solved. I have included an instruction in the 'Readme' file to run `bundle exec rails webpacker:install`, just to make sure webpacker creates the manifest file. Run this after running `bundle install` when you clone the repo locally and everything should be fine.

**Review by rOluochKe — APPROVED** [review]
Hello @RamseyNjire 
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Comment by RamseyNjire** [comment]
Great, doing the merge! All good here.

**Pull request merged**